simtools/util/general.py

```diff
@@ -165,75 +165,75 @@ def _validateAndConvertValue_without_units(value, valueKeys, parName, parInfo):
     if all([isinstance(v, str) for v in value]):
-        # In case values are string, e.g. mirrorNumbers = 'all'
+        # In case values are string, e.g. mirror_numbers = 'all'
         # This is needed otherwise the elif condition will break
         pass
     elif any([u.Quantity(v).unit != u.dimensionless_unscaled for v in value]):
-        msg = "Config entry {} should not have units".format(parName)
+        msg = "Config entry {} should not have units".format(par_name)
         logger.error(msg)
         raise InvalidConfigEntry(msg)
 
-    if valueKeys:
-        return {k: v for (k, v) in zip(valueKeys, value)}
-    return value if len(value) > 1 or undefinedLength else value[0]
+    if value_keys:
+        return {k: v for (k, v) in zip(value_keys, value)}
+    return value if len(value) > 1 or undefined_length else value[0]
 
 
-def _checkValueEntryLength(value, parName, parInfo):
+def _check_value_entry_length(value, par_name, par_info):
     """
     Validate length of user input parmeters
 
     Parameters
     ----------
     value: list
         list of user input values
-    parName: str
+    par_name: str
         name of parameter
-    parInfo: dict
+    par_info: dict
         dictionary with parameter info
 
     Returns
     -------
-    valueLength: int
+    value_length: int
         length of input list
-    undefinedLength: bool
+    undefined_length: bool
         state of input list
 
     """
     logger = logging.getLogger(__name__)
 
     # Checking the entry length
-    valueLength = len(value)
-    logger.debug("Value len of {}: {}".format(parName, valueLength))
-    undefinedLength = False
+    value_length = len(value)
+    logger.debug("Value len of {}: {}".format(par_name, value_length))
+    undefined_length = False
     try:
-        if parInfo["len"] is None:
-            undefinedLength = True
-        elif valueLength != parInfo["len"]:
-            msg = "Config entry with wrong len: {}".format(parName)
+        if par_info["len"] is None:
+            undefined_length = True
+        elif value_length != par_info["len"]:
+            msg = "Config entry with wrong len: {}".format(par_name)
             logger.error(msg)
             raise InvalidConfigEntry(msg)
     except KeyError:
-        logger.error("Missing len entry in parInfo")
+        logger.error("Missing len entry in par_info")
         raise
 
-    return valueLength, undefinedLength
+    return value_length, undefined_length
 
 
-def _validateAndConvertValue_with_units(value, valueKeys, parName, parInfo):
+def _validate_and_convert_value_with_units(value, value_keys, par_name, par_info):
     """
     Validate input user parameter for input values with units.
 
     Parameters
     ----------
     value: list
        list of user input values
-    valueKeys: list
+    value_keys: list
        list of keys if user input was a dict; otherwise None
-    parnName: str
+    parn_name: str
```

Comment:
There is a typo here. should be par_name as in the function declaration

Reply:
Fixed